src/main/resources/static/schema.graphql

```diff
@@ -11,6 +11,7 @@ type Query {
     getOrderByOrderNumber(orderNumber: String): Order
     #item
     getItems(categoryId: Int, regions: [RegionType], searchString: String, page: Int, size: Int, sort: [String]): ItemPageInfo
+    getItemByName(itemName: String): Item
```

Comment:
Please change "itemName" field to be required to be consistent with corresponding REST API.